docs/index.rst

```diff
@@ -120,6 +120,15 @@ Extensions
       :padding: 1
 
       Adds the "last updated" date at the bottom of each documentation page (obtained from the Git commit date).
+      
+   .. grid-item-card:: sphinxext-opengraph
+      :img-top: _static/sphinxext-opengraph.png
+      :link: https://github.com/wpilibsuite/sphinxext-opengraph
+      :padding: 1
+
+      Sphinx extension to generate `Open Graph metadata <https://ogp.me/>`_ ✨ for each page of your documentation.
```

Comment:
```suggestion
      Generates Open Graph metadata ✨ for each page of your documentation.
```

* I removed the "Sphinx extension" since all of them are Sphinx extensions
* Removed the link since it does not work because the whole element is a link itself